addons/payment_transfer/models/payment_acquirer.py

```diff
@@ -19,6 +19,11 @@ def _get_providers(self, cr, uid, context=None):
         providers.append(['transfer', 'Wire Transfer'])
         return providers
 
+    _defaults = {
+        'provider': 'transfer',
```

Comment:
well... if I have Paypal and wire transfer, you are forcing wire transfer as default... why ?